tests/sampleTests/replaySample.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { expect, test } from '@playwright/test';
+import { Polly } from '@pollyjs/core';
+
+import { submitPaymentResult } from '../api.mock';
+import { replayInitializer } from '../polly.global.setup';
+
+// Uncommmnet to continue in a headed browser
+// test.use({
+//     headless: false,
+//     viewport: { width: 1000, height: 1000 },
+// });
+
+test.describe('Replay mode sample', () => {
+
+    let polly: Polly;
+
+    test.beforeEach(async ({ page }) => {
+        // 1. Setup PollyJS
+        polly = await replayInitializer({
+            playwrightContext: page,
+            recordingName: 'sample',
+            storeURL: 'https://my-dev-store-745516528.store.bcdev',
```

Comment:
Can this be passed as environment variable?

Reply:
This is for matching URLs in a HAR.
I'll try to get it automatically from a HAR so that no manual input is required.